### ISSUE TITLE
Build just the editor as VS Code extension

### ISSUE THREAD

**Issue by jasonycw** [haystackeditor/haystack-editor#6]
The editor looks very promising! 

However, downloading a full new IDE to checkout the editor is quite annoying. 

It will be great if just the editor will limited set of feature be implemented as an VS Code extension for people to try out.

**Comment by hereisHandesu** [comment]
I agree with it! There are many potential problems: some shortcut keys doesn't work, and part of my extension do not function in the new IDE.
But I had never seen such a brilliant idea that edit codes as node and map, I am really fond of it!

**Comment by LordCasser** [comment]
I agree with it too.

**Comment by EddieDover** [comment]
I'm pretty sure that'll never happen because they want that sweet Enterprise money.

**Comment by autonull** [comment]
https://github.com/automenta/spacegraphjs

"Adapt this zooming web canvas HTML/JS app into a VisualStudio code extension that embeds code editor graph nodes in a zoomable canvas."

https://aistudio.google.com/app/prompts?state=%7B%22ids%22:%5B%221Fzyzk8J34nvi3huIkn7e_dwsFbneWxXm%22%5D,%22action%22:%22open%22,%22userId%22:%22105334578998910226747%22,%22resourceKeys%22:%7B%7D%7D

Anyone interested in doing this?
